docs/plugins/windows_event.md

```diff
@@ -0,0 +1,56 @@
+# `windows_event` plugin
+
+The `windows_event` plugin consumes Windows Event log entries from the local filesystem and outputs parsed entries.
```

Comment:
Logs are read from windows event viewer API, not filesystem.

Reply:
updated description